hawkular-inventory-itest-parent/hawkular-inventory-itest-dist/pom.xml

```diff
@@ -0,0 +1,592 @@
+  </properties>
+
+  <build>
+    <plugins>
+      <!-- skip tests -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <skipTests>true</skipTests>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>hawkular-inventory-itest</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <!-- test dist dependencies -->
+    <dependency>
+      <groupId>org.hawkular.bus</groupId>
+      <artifactId>hawkular-bus-broker-wf-extension</artifactId>
+      <version>${test-dist-version.org.hawkular.bus}</version>
+      <classifier>module</classifier>
+      <type>zip</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.bus</groupId>
+      <artifactId>hawkular-bus-ra-wf-module</artifactId>
+      <version>${test-dist-version.org.hawkular.bus}</version>
+      <classifier>module</classifier>
+      <type>zip</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.nest</groupId>
+      <artifactId>hawkular-nest-wf-extension</artifactId>
+      <version>${test-dist-version.org.hawkular.bus}</version>
+      <classifier>module</classifier>
+      <type>zip</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>hawkular-inventory-dist</artifactId>
+      <version>${test-dist-version.org.hawkular.inventory}</version>
+      <type>war</type>
+    </dependency>
+    <!--<dependency>-->
+      <!--<groupId>org.hawkular.cmdgw</groupId>-->
+      <!--<artifactId>hawkular-command-gateway-war</artifactId>-->
+      <!--<version>${version.org.hawkular.cmdgw}</version>-->
+      <!--<type>war</type>-->
+    <!--</dependency>-->
+    <!--<dependency>-->
+      <!--<groupId>org.hawkular.metrics</groupId>-->
+      <!--<artifactId>hawkular-metrics-api-jaxrs</artifactId>-->
+      <!--<version>${version.org.hawkular.metrics}</version>-->
+      <!--<type>war</type>-->
+    <!--</dependency>-->
+    <dependency>
+      <groupId>org.hawkular.commons</groupId>
+      <artifactId>hawkular-commons-embedded-cassandra-ear</artifactId>
+      <version>${test-dist-version.org.hawkular.commons}</version>
+      <type>ear</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.accounts</groupId>
+      <artifactId>hawkular-accounts</artifactId>
+      <version>${test-dist-version.org.hawkular.accounts}</version>
+      <type>war</type>
+    </dependency>
+    <!--<dependency>-->
+      <!--<groupId>org.hawkular.accounts</groupId>-->
+      <!--<artifactId>hawkular-accounts-events-backend</artifactId>-->
+      <!--<version>${version.org.hawkular.accounts}</version>-->
+      <!--<type>war</type>-->
+    <!--</dependency>-->
```

Comment:
Remove `hawkular-accounts-events-backend` altogether if the distro works without it.